### PR TITLE
/docs/app-dev formatting fixes

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

/docs/app-dev formatting fixes

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/cloudtrail.md [hunk 1]
```diff
@@ -277,7 +277,7 @@ See information about CloudTrail console logins, including location of users, lo
 
 **Outlier - Success Login**. Shows outliers in count of successful logins on a line chart with threshold for the last 24 hours.
 
-**Login Results - One Day Time Compare. **Shows the number of login successes and failures in an aggregation table, that compares the current count with the count from one day ago.
+**Login Results - One Day Time Compare.** Shows the number of login successes and failures in an aggregation table, that compares the current count with the count from one day ago.
 
 **Outlier - Failed Login**. Shows outliers in count of failed login attempts on a line chart with threshold for the last 24 hours.
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/cloudtrail.md:283]: This file needs more asterisks cleanup. Search for ** .

File: docs/integrations/amazon-aws/dynamodb.md [hunk 2]
```diff
@@ -156,8 +156,6 @@ Enter a parse expression to create an “account” field that maps to the alias
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/dynamodb.md:156]: This file needs more asterisks cleanup. Search for ** .

File: docs/integrations/amazon-aws/elasticache.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ account={{account}} region={{region}} namespace={{namespace}} "\"eventSource\":\
 1. To your Hosted Collector, add an [AWS CloudTrail Source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md).
    * **Name**. Enter a name to display for the new Source.
    * **Description**. Enter an optional description.
-   * **S3 Region**. Select the Amazon Region for your** ElastiCache** S3 bucket.
+   * **S3 Region**. Select the Amazon Region for your **ElastiCache** S3 bucket.
    * **Bucket Name**. Enter the exact name of your **ElastiCache** S3 bucket.
    * **Path Expression**. Enter the string that matches the S3 objects you'd like to collect. You can use a wildcard (*) in this string. (DO NOT use a leading forward slash. See [Amazon Path Expressions](/docs/send-data/hosted-collectors/amazon-aws/amazon-path-expressions).) The S3 bucket name is not part of the path. Don’t include the bucket name when you are setting the Path Expression.
    * **Source Category**. Enter aws/observability/cloudtrail/logs
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/elasticache.md:125]: This file needs more asterisks cleanup. Search for ** .

File: docs/integrations/app-development/jfrog-xray.md [hunk 4]
```diff
@@ -291,7 +291,7 @@ To run the function manually, do the following:
    ```bash
    python3 -m sumojfrogxray.main
    ```
-2. The script generates logs in **/tmp/sumoapiclient.log **by default. Check these logs to verify whether the script is getting triggered or not.
+2. The script generates logs in **/tmp/sumoapiclient.log** by default. Check these logs to verify whether the script is getting triggered or not.
 3. If you get an error such as unable to execute `'gcc': No such file or directory, error: command 'gcc' failed with exit status 1`, go to your AWS EC2 instance and run the following commands:
 ```bash
 sudo yum -y install gcc
```
Comment (jpipkin1) [inline docs/integrations/app-development/jfrog-xray.md:297]: This file needs more asterisks cleanup. Search for ** .

File: docs/integrations/saas-cloud/slack.md [hunk 5]
```diff
@@ -308,7 +308,7 @@ Use this dashboard to:
 
 ### Audit Overview Dashboard
 
-The **Slack - Audit Overview **dashboard provides details around  Slack audit actions, and trends.
+The **Slack - Audit Overview ** dashboard provides details around  Slack audit actions, and trends.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/slack.md:312]: This file needs more asterisks cleanup. Search for ** .

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 6]
```diff
@@ -179,15 +179,15 @@ Use this dashboard to:
...
+**F5 - BIG-IP LTM - Outlier Analysis ** dashboard provides an at-a-glance view of TCP connection and login request outliers, along with network traffic and Traffic Manager resource usage.
 
 **Use this dashboard to** find any outliers in your incoming traffic, CPU and memory usage.
 
 <img src={useBaseUrl('img/integrations/security-threat-detection/F5-BIGIPLTM-OutlierAnalysis.png')} alt="F5 dashboards" />
 
 ### User Session Events
 
-**F5 - BIG-IP LTM - User Session Events **dashboard provides an at-a-glance view of user activity and their geographic location. This dashboard also provides information on login trends, operating systems used, user session IDs, and users by their application profile.
+**F5 - BIG-IP LTM - User Session Events ** dashboard provides an at-a-glance view of user activity and their geographic location. This dashboard also provides information on login trends, operating systems used, user session IDs, and users by their application profile.
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:191]: This file needs more asterisks cleanup. Search for ** .

File: docs/api/dashboard-data.md [hunk 7]
```diff
@@ -93,7 +93,7 @@ curl --trace-ascii - --user EMAIL:PASSWORD -H 'Accept: application/json' "https:
 ...
 ```
 
-The result is a JSON object with one key: **dashboards**. The value for the key is an array of dashboard definitions. In this example, the dashboard definition for the first entry in the array is:
+The result is a JSON object with one key: ** dashboards**. The value for the key is an array of dashboard definitions. In this example, the dashboard definition for the first entry in the array is:
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/api/dashboard-data.md:97]: thank you!! wow, good eye

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**